src/lib/restapi/training.ts

```diff
@@ -83,6 +86,26 @@ async function getTraining(req: auth.RequestWithProject, res: Express.Response)
 }
 
 
+function getTrainingItem(req: auth.RequestWithProject, res: Express.Response) {
+    if (req.project.type !== 'imgtfjs') {
+        return res.status(httpstatus.BAD_REQUEST).json({ error : 'Only supported for image projects' });
+    }
+
+    return visrec.getTrainingItemData(req.project, req.params.trainingid)
```

Comment:
Need to add a version of this that can be invoked using scratchkeys, so that a new model can be trained from Scratch